AutomatedTesting/Gem/PythonTests/atom_renderer/atom_utils/atom_component_helper.py

```diff
@@ -0,0 +1,19 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project. For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+
+File to assist with common hydra component functions or constants used across various Atom tests.
+"""
+
+# Light type options for the Light component.
+LIGHT_TYPES = [
+    0,  # UNKNOWN
+    1,  # SPHERE
+    2,  # SPOTDISK
+    3,  # CAPSULE
+    4,  # QUAD
+    5,  # POLYGON
+    6,  # SIMPLEPOINT
+    7,  # SIMPLESPOT
+]
```

Comment:
Having an array like this is pretty useless because LIGHT_TYPES[n] is always equal to n. Much more useful would be a dictionary of some kind so you could do something like LIGHT_TYPES["sphere"] and get back 1 for example. Then you also wouldn't need to comment all the places where you're using LIGHT_TYPE with what type it actually is, and if indices change in the future it'll be much easier to fix.

Reply:
I tried to use it as an `Enum` before but hydra doesn't play nice with `Enum` however a `dict` is probably better than a `list` for this ill update it

Reply:
Looks good